src/app/modules/+badges/components/last-achievements/last-achievements.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <div class="header">
+    <p class="text-uppercase">last achievements</p>
```

Comment:
add empty lines between tags on the same level

Reply:
and please check everywhere

Reply:
fixed